homework/style.css

```diff
@@ -1,3 +1,127 @@
 .alert-error {
```

Comment:
On a mobile device you are not making full use of the screen width:

![image](https://user-images.githubusercontent.com/2788771/65896814-3ab0f100-e3ae-11e9-956d-0e3a81c1599b.png)